client/dist/index.html

```diff
@@ -5,16 +5,6 @@
   <body>
     <div id="app"></div>
     <style>
```

Comment:
The CSS looks good to me. I'm curious, is there a particular reason why you added the style tag inside the body tag and not somewhere else like the head, or why you just didn't create a completely separate style folder? I'm wondering if you had to do that because of react-modal or something.

Reply:
For me using the style tag is just a place holder. I may have separate style sheets for each module or just push it all onto one style sheet. This was just a very quick way to affect everything in the body.